python/lsst/jointcal/jointcal.py

```diff
@@ -359,11 +359,6 @@ def __init__(self, butler=None, profile_jointcal=False, **kwargs):
         # To hold various computed metrics for use by tests
         self.job = Job.load_metrics_package(subset='jointcal')
 
-    # We don't need to persist config and metadata at this stage.
-    # In this way, we don't need to put a specific entry in the camera mapper policy file
-    def _getConfigName(self):
-        return None
-
```

Comment:
Do you want to add in a comment that we don't need to persist metadata at this stage?